### ISSUE TITLE
<link media> queries are stripped

### ISSUE THREAD

**Issue by bennypowers** [divriots/jampack#61]
before jampack:

```html
<link rel="stylesheet" media="(prefers-color-scheme: light)" href="/assets/theme-light.css">

<link rel="stylesheet" media="(prefers-color-scheme: dark)" href="/assets/theme-dark.css">
```

after jampack
```html
<link rel="stylesheet" media="" href="/assets/theme-light.css">

<link rel="stylesheet" media="" href="/assets/theme-dark.css">
```

**Comment by georges-gomes** [comment]
Wait what!? This is annoying. I will check this out shortly.

**Comment by georges-gomes** [comment]
This is now fixed in version `0.11.+`
I had to upgrade `cheerio`

**Comment by bennypowers** [comment]
I still have this problem with 0.11.2

```
❯ npx npm-why cheerio                                                                                                                                             (base) 

  Who required cheerio:

  bennypowers.dev > cheerio@1.0.0-rc.12
  bennypowers.dev > eleventy-plugin-nesting-toc > cheerio@1.0.0-rc.12
  bennypowers.dev > eleventy-plugin-slide-decks > cheerio@1.0.0-rc.12



bennypowers.dev on  main [$!] via  v18.16.0 took 3s 
❯ npx npm-why @divriots/cheerio                                                                                                                                   (base) 

  Who required @divriots/cheerio:

  bennypowers.dev > @divriots/jampack > @divriots/cheerio@1.0.0-rc.12


❯ cat _site/posts/import-map-cdn/index.html | grep media=                                                                                                         (base) 
    <link href="/assets/theme-light.css" rel="stylesheet" media="">
    <link href="/assets/theme-dark.css" rel="stylesheet" media="">

```

**Comment by georges-gomes** [comment]
Sorry I probably tested badly.
This is after minification of HTML.
This is coming from `html-minify-terser` apparently.

Looking at it. Thanks

**Comment by georges-gomes** [comment]
I was compressing the media queries with CSS 🤦‍♂️
Fixed (for real) in `0.11.3`

Thanks again @bennypowers !

**Comment by bennypowers** [comment]
fix confirmed!

```
bennypowers.dev on  main [$!] via  v18.16.0 took 1m33s 
❯ cat _site/posts/import-map-cdn/index.html | grep media=                                                                                                         (base) 
    <link href="/assets/theme-light.css" rel="stylesheet" media="(prefers-color-scheme: light)">
    <link href="/assets/theme-dark.css" rel="stylesheet" media="(prefers-color-scheme: dark)">

```

Nice! merci, georges
